src/Compiler/Microsoft.AspNetCore.Razor.Language/src/ChecksumUtilities.cs

```diff
@@ -12,9 +11,9 @@ namespace Microsoft.AspNetCore.Razor.Language;
 
 internal static class ChecksumUtilities
 {
-    public static string BytesToString(byte[] bytes)
+    public static string BytesToString(ImmutableArray<byte> bytes)
```

Comment:
Did you consider using `ReadOnlySpan<byte>`?

Reply:
We could, but it really doesn't buy us anything at the moment. All the callsites are just passing the result of `GetChecksum()` from Roslyn, which is `ImmutableArray<byte>`. We're not avoiding any intermediaries with a ROS here.